source/includes/fundamentals/code-snippets/records/RecordAnnotationExample.java

```diff
@@ -0,0 +1,30 @@
+package fundamentals.codecs.records;
+
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+import com.mongodb.client.result.InsertOneResult;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RecordAnnotationExample {
+    public static void main(String[] args) {
+        // Replace the uri string with your MongoDB deployment's connection string
+        String uri = "<your connection uri>";
+        try (MongoClient mongoClient = MongoClients.create(uri)) {
+
+            MongoDatabase database = mongoClient.getDatabase("sample_data");
+            MongoCollection<NetworkDeviceRecord> collection = database.getCollection("network_devices", NetworkDeviceRecord.class);
+
+            // insert the document
+            collection.insertOne(new NetworkDeviceRecord("fedcba09876543210abcdef0", "Enterprise Wi-fi", "router"));
```

Comment:
I would show this as `new ObjectId().toHexString()` instead of `"fedcba09876543210abcdef0"`.   Because where would you get that string from if not an ObjectId instance?